TestCases/serial_regression.py

```diff
@@ -490,7 +490,7 @@ def main():
     centrifugal_stage.cfg_dir   = "turbomachinery/centrifugal_stage"
     centrifugal_stage.cfg_file  = "centrifugal_stage.cfg"
     centrifugal_stage.test_iter = 100
-    centrifugal_stage.test_vals = [-10.166364, 1.621172, 2.206476e+01, 5.271075e-01] #last 4 columns
+    centrifugal_stage.test_vals = [-10.166717, 1.620590, 30.003500, 0.487959] #last 4 columns
```

Comment:
Thanks. Yes, the quantities for the turbomachinery test case are different now. There is a completely new structure for the turbomachinery performance that will be integrated soon when we will merge the turbomachiney features. After discussing about this with Salvo (that prepared that regression test), we decided that it's OK to just change the residuals for the time being. I can, anyway, have a closer look.